gio/src/app_info.rs

```diff
@@ -80,7 +80,7 @@ impl<O: IsA<AppInfo>> AppInfoExtManual for O {
         unsafe {
             ffi::g_app_info_launch_uris_async(
                 self.as_ref().to_glib_none().0,
-                uris.to_glib_none().0,
+                uris.as_ref().to_glib_none().0,
```

Comment:
This has a normal `to_glib_none()` implementation now right? This change wasn't necessary in auto code, and returns the slice like an identity operator:

https://doc.rust-lang.org/std/primitive.slice.html#impl-AsRef%3C%5BT%5D%3E